internal/pkg/source/source.go

```diff
@@ -413,10 +415,15 @@ func isSourceType(c classifier, t types.Type) bool {
 		key := isSourceType(c, tt.Key())
 		elem := isSourceType(c, tt.Elem())
 		return key || elem
-	case *types.Basic, *types.Interface, *types.Tuple, *types.Struct:
+	case *types.Basic, *types.Struct, *types.Tuple, *types.Interface, *types.Signature, *types.Pointer:
```

Comment:
Good point.  I was trying to be a competionist, but that does seem a bit pathological in hindsight.

Reply:
I ended up remaining a completionist, since `*types.Pointer` is the _only_ type that wasn't otherwise included.  I've made it clear that it should be unreachable.  On the one hand, it's introducing dead code.  On the other hand, it feels like an omission or possible accident to leave it out.  So I could go either way on it, really.